src/components/datePicker/tests/DatePickerDropdown.spec.tsx

```diff
@@ -56,7 +56,14 @@ describe('Date picker', () => {
       expect(datesSelectionBoxesProps).toEqual(DATE_PICKER_DROPDOWN_BASIC_PROPS.datesSelectionBoxes);
     });
 
-    it('should display a <DatePickerBox />', () => {
+    it('should not display a <DatePickerBox /> if it is not opened', () => {
+      expect(datePickerDropdown.find('DatePickerBox').length).toBe(0);
+    });
+
+    it('should display a <DatePickerBox /> if it is opened', () => {
+      let propsIsOpen: IDatePickerDropdownProps = _.extend({}, DATE_PICKER_DROPDOWN_BASIC_PROPS, { isOpened: true });
```

Comment:
Whole lot of nitpick, but the actual props is called isOpened, it should be the same in tests. (applies to the other ones too)